vaadin-text-field-flow-parent/vaadin-text-field-flow/src/test/java/com/vaadin/flow/component/textfield/tests/NumberFieldValidationTest.java

```diff
@@ -0,0 +1,108 @@
+package com.vaadin.flow.component.textfield.tests;
+
+import com.vaadin.flow.component.shared.ValidationError;
+import com.vaadin.flow.component.textfield.NumberField;
+import com.vaadin.flow.data.binder.Binder;
+import com.vaadin.flow.data.binder.BindingValidationStatusHandler;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Objects;
+
+public class NumberFieldValidationTest {
```

Comment:
**suggestion:** I would suggest we mention `Binder` in the test's class name considering that it only focuses on binder validation and doesn't test the default constraint validation.

```java
public class NumberFieldBinderValidationTest {
  ...
}
```

Reply:
I moved them to a new package, so it will be a bit more organized.